web/thesauruses/java/17/operators.json

```diff
@@ -118,6 +118,34 @@
       "code": ">>",
       "name": "Right shift bitwise operator"
     },
+    "bitwise_and": {
+      "code": "&",
```

Comment:
Since some languages put operands in different places (like C++ is `a & b` but lisp is `(logand a b)`, can you put where where the operands would be?

(I realize the original file didn't do this but I'd like the files moving forward to do this.)